### DISCUSSION TITLE
Log/Events retention in Security Onion 2.4.80

### DISCUSSION THREAD

**Discussion by networklord** [Security-Onion-Solutions/securityonion#13552] [General] [answered]
Hello all,

By default for how long SO is keeping records like PCAP, alarm events etc? I have installed Standalone on 200GB HDD. Could not find option to control retention in configuration

Thanks in Advance

**Comment by dougburks** [comment, marked as answer]
By default, Security Onion will try to store as much data as possible without filling up your storage. For more information, please see:
https://docs.securityonion.net/en/2.4/stenographer.html#disk-free-percentage
https://docs.securityonion.net/en/2.4/elasticsearch.html#index-management

**Reply by networklord** [reply]
Thanks very much!
